docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,629 @@
+
+To find out what has changed in your local copy of your feature branch, type :
+
+```
+git status
+```
+
+In my case I have :
+
+![submit_pr_staging_1.png](./imgs/submit_pr_staging_1.png)
+
+There is a new file `docs/site/submitting_a_pr.md` that is `untracked` and needs to be `staged`.
+
+To stage all untracked files, you can type:
+
+```
+git add --all
+```
+
+To stage one untracked file at a time, you can type:
+
+```
+git add {relative path to file from root directory}
+```
+
+In my case, this would be
+
+```
+git add docs/site/submitting_a_pr.md
+```
+
+To see which files are `staged` and `unstaged`, type:
+
+```
+git status
+```
+![submit_pr_staging_2.png](./imgs/submit_pr_staging_2.png)
+
+In my case, the file `docs/site/submitting_a_pr.md` is properly `staged`, and there are no `unstaged (untracked)` files.
+
+### 13. Committing your changes
+
+When you are pleased with your staged changes, it is time to create a `commit` and give it a message.
+
+Please read the [Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines) guidelines to correctly format your commit message.
+
+To help with abiding by the rules of commit messages, please use the `commitizen` tool mentioned in the documentation above. This means will we use `git cz` instead of `git commit`.
+
+In my case, I want the commit message to look like this:
+
+```
+docs: tutorial on how to submit a pull request to lb4
+
+This tutorial shows contributors step-by-step instructions on how to submit a pull request (PR) to Loopback v4
+
+```
+
+If you need the long description to span multiple lines, use `\n` characters to separate the sentences.
+
+Let's go through the interactive prompts of `commitizen`.
+
+Type
+```
+git cz
+```
+
+The first prompt asks you to choose the `type` value:
+
+![submit_pr_git_cz_1.png](./imgs/submit_pr_git_cz_1.png)
+
+
+Using the `down arrow` key, I will choose `docs` and press `ENTER`.
+
+![submit_pr_git_cz_2.png](./imgs/submit_pr_git_cz_2.png)
+
+The next prompt asks you to choose the `scope` value:
+
+![submit_pr_git_cz_3.png](./imgs/submit_pr_git_cz_3.png)
+
+In my case, I won't be specifying a scope value, so I will simply press `ENTER` to skip.
+
+The next prompt asks you to specify the `short description` value:
+
+![submit_pr_git_cz_4.png](./imgs/submit_pr_git_cz_4.png)
+
+I specify : `tutorial on how to submit a pull request to lb4` .
+
+![submit_pr_git_cz_5.png](./imgs/submit_pr_git_cz_5.png)
+
+The next prompt asks you to specify the `long description` value:
+
+![submit_pr_git_cz_6.png](./imgs/submit_pr_git_cz_6.png)
+
+I specify : `This tutorial shows contributors step-by-step instructions on how to submit a pull request (PR) to Loopback v4`
+
+![submit_pr_git_cz_7.png](./imgs/submit_pr_git_cz_7.png)
+
+
+The next prompt asks if any `breaking changes` will be introduced.
+
+![submit_pr_git_cz_8.png](./imgs/submit_pr_git_cz_8.png)
```

Comment:
I like this prompts that forces people rethink if their code contain breaking change.